pkgs/applications/networking/calls/default.nix

```diff
@@ -80,7 +88,8 @@ stdenv.mkDerivation rec {
     "-Dgtk_doc=true"
   ];
 
-  doCheck = true;
+  # https://gitlab.gnome.org/GNOME/calls/-/issues/258
```

Comment:
So the tests are disabled until that issue is resolved?

Reply:
Yes. I've updated the comment to mention that.